### PR TITLE
added darkmode to website

### LINKED ISSUE #181: Adding Dark Mode to The Website
### **Feature request**:
I would love to add a dark mode to this website which will enhance the website.

### PR DESCRIPTION
**Pull request by arpit456jain**
# Description:

added darkmode to website

Fixes #181 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

 
# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

# Screenshots / Video:

https://user-images.githubusercontent.com/55352601/118792615-72d3b300-b8b5-11eb-89ad-efe6431f5a3b.mp4

### CODE REVIEW

**Comment by arpit456jain** [comment]
@SaiSudhaV @sruti2024

**Comment by sruti2024** [comment]
can u pls add this to all the pages?

**Comment by arpit456jain** [comment]
> can u pls add this to all the pages?

I think only one page is left...
That view projects details

If other than this any page is left please let me know @sruti2024 
And i will add dark mode to it in an hour so please try to review it again after 1 hour

**Comment by arpit456jain** [comment]
@sruti2024 @SaiSudhaV

**Review by SaiSudhaV — APPROVED** [review]

**Comment by arpit456jain** [comment]
@sruti2024 please merge this

**Comment by Aryamanz29** [comment]
Hii 👋@arpit456jain 
Please add internal CSS to their respective external CSS files

File: templates/index.html [hunk 1]
```diff
@@ -3,7 +3,19 @@
 {% load static %}
 {% block title %}PRO_Act || Home{% endblock title %}
 {% block body %}
-
+<style>
```
Comment (Aryamanz29) [inline templates/index.html:6]: Moves this to `static/css/index.css`

File: templates/modules.html [hunk 2]
```diff
@@ -14,15 +14,121 @@
   <link rel="stylesheet" href="{% static 'css/moduleStyle.css' %}" type="text/css">
   <title>PRO_Act || Modules</title>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js" type="text/javascript"></script>
+  <style>
```
Comment (Aryamanz29) [inline templates/modules.html:17]: Same here, move this to `static/css/moduleStyle.css`

File: templates/profile.html [hunk 3]
```diff
@@ -4,6 +4,14 @@
 {% block title %}PRO_Act | Profile{% endblock title %}
 
 {% block body %}
+<style>
```
Comment (Aryamanz29) [inline templates/profile.html:7]: Same comment here

File: templates/project_add.html [hunk 4]
```diff
@@ -4,7 +4,14 @@
 
 {% block body %}
 <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
-
```
Comment (Aryamanz29) [inline templates/project_add.html:7]: Create new for this one` static/css/project_add.css`

**Pull request merged**